src/table/context/TableContext.tsx

```diff
@@ -37,7 +38,9 @@ export const TableContextProvider = ({
   tableWidth = 0,
 }: ContextProviderProps) => {
   const [headRowHeight, setHeadRowHeight] = useState(0);
-  const [focusedCellCoord, setFocusedCellCoord] = useState((cellCoordMock || [0, 0]) as [number, number]);
+  const [focusedCellCoord, setFocusedCellCoord] = useState(
```

Comment:
`[0, 0]` caused an extra render to occur every time the user changed page

Reply:
nice catch